src/components/auction/OrderPlacement/index.tsx

```diff
@@ -262,6 +264,27 @@ const OrderPlacement: React.FC<OrderPlacementProps> = (props) => {
   ])
   const notApproved = approval === ApprovalState.NOT_APPROVED || approval === ApprovalState.PENDING
   const orderPlacingOnly = auctionState === AuctionState.ORDER_PLACING
+  const coversClearingPrice = (price: string | undefined, showPriceInverted: boolean): boolean => {
+    const standardizedPrice = showPriceInverted
+      ? getInverse(Number(price), NUMBER_OF_DIGITS_FOR_INVERSION).toString()
+      : price
```

Comment:
This could by part of the function [`convertPriceIntoBuyAndSellAmount`](https://github.com/gnosis/ido-ux/blob/d1381ced440f950a0f5970c112b94ce1c4d925b3/src/utils/prices.ts#L15), and return a `standardizedPrice`, but is a trade off. 
Advantage:
✅ Repeated code would be eliminated and clean up responsibility in the hooks/[`useOrderPlacementState`](https://github.com/gnosis/ido-ux/blob/d1381ced440f950a0f5970c112b94ce1c4d925b3/src/state/orderPlacement/hooks.ts#L200) as well as in this `OrderPlacement` component.

Disadvantage:
❌ It would add more complexity to the function [`convertPriceIntoBuyAndSellAmount`](https://github.com/gnosis/ido-ux/blob/d1381ced440f950a0f5970c112b94ce1c4d925b3/src/utils/prices.ts#L15).

Reply:
Personally, I try to keep functions simple, until we have 3 times the same pattern. If there are 3 examples, removing code repetition is more advantageous...

Anyways... it's up to you! All fine

Reply:
I will keep an eye on it in case it becomes necessary🕵️.